src/components/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { Tabs } from '.';
+import { Box } from '@mui/material';
+
+const meta = {
+	title: 'Other/Tabs',
+	component: Tabs,
+	tags: ['autodocs'],
+} satisfies Meta<typeof Tabs>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+	args: {
+		tabs: [
+			{ label: 'one', render: <Box>Rendering the first tab</Box> },
+			{ label: 'two', render: <Box>Rendering the second tab</Box> },
+			{ label: 'three', render: <Box>Rendering the third tab</Box> },
```

Comment:
Might be useful to show how to pass props in the story

```suggestion
			{ label: 'three', render: <Box>Rendering the third tab</Box> },
			{
				label: 'four',
				render: <Box>Not rendering the fourth tab (disabled)</Box>,
				tabProps: { disabled: true },
			},
```